website/_config.yml

```diff
@@ -0,0 +1,19 @@
+title: Krayon
+description: Cross platform (Android, iOS & JavaScript) canvas library.
+
+plugins:
+  - jekyll-remote-theme
+  - jekyll-spaceship
+
+remote_theme: jekyll/minima
+
+kramdown:
+  parse_block_html: true
+
+jekyll-spaceship:
+  processors:
+    - mermaid-processor
+  mermaid-processor:
+    mode: pre-fetch
```

Comment:
`default` mode causes diagrams to be huge. 🤷
Upstream issue: https://github.com/jeffreytse/jekyll-spaceship/issues/87